src/pages/responsible/_securing-account.js

```diff
@@ -1,55 +1,78 @@
 import React from 'react'
 import styled from 'styled-components'
 import { Container, Flex, Box } from 'components/containers'
-import { Header, Text } from 'components/elements'
-import { localize } from 'components/localization'
+import { Header, LocalizedLinkText, Text } from 'components/elements'
+import { localize, Localize } from 'components/localization'
 import device from 'themes/device'
 import InstallAntivirus from 'images/svg/responsible/install-latest-antivirus.svg'
 import DifferentPassword from 'images/svg/responsible/diff-password.svg'
 import PaymentMethod from 'images/svg/responsible/payment-method.svg'
 import SecureBrowser from 'images/svg/responsible/secure-browser.svg'
 
+const StyledBox = styled(Box)`
+    @media ${device.tablet} {
+        padding-top: 40px;
+    }
+`
+const StyledContainer = styled(Container)`
+    @media ${device.tablet} {
+        width: 100%;
+        padding: 0 16px;
+    }
+`
```

Comment:
Let's use the responsive props for adding these styles.

Reply:
I've tried it before,but still not working..

Reply:
fixed in latest commit